django/db/models/fields/tuple_lookups.py

```diff
@@ -233,6 +240,23 @@ def check_rhs_elements_length_equals_lhs_length(self):
                 f"must have {len_lhs} elements each"
             )
 
+    def check_rhs_is_query(self):
+        if not (isinstance(self.rhs, Query)):
+            lhs_str = self.get_lhs_str()
+            raise ValueError(
+                f"{self.lookup_name!r} subquery lookup of {lhs_str} "
+                "must be a Query object"
+            )
```

Comment:
We should probably include something specific about `self.rhs` in this error message. Currently we only have `self.lhs`.

Reply:
Alright, I added the class name to the error message 👍